.github/workflows/trigger-v1-canary.yml

```diff
@@ -0,0 +1,13 @@
+name: trigger v1 canary release
+
+on:
+  schedule:
+    # At 4 AM UTC every day
+    - cron: 0 4 * * *
```

Comment:
[pp] Also, the canary release should run before the tests as the smoke tests use the canary version for testing.